### ISSUE TITLE
How to generate sources for an xsd schema including another schema? Broken wiki links.

### ISSUE THREAD

**Issue by samodadela** [highsource/jaxb-tools#539]
Hi, 

I have two xsd files. `Main.xsd` an `Common.xsd`. The files are local (not available via HTTP URLs).

`Main.xsd` includes `Common.xsd`:
```
Main.xsd:

<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified">
    <xs:include schemaLocation="Common.xsd"/>
...
```

Can you please point me to a working sample project that generates classes for such a case?

My preferences would be:
- To have also the binding (xjb) files separate. Can this be done? How?
- Put all the classes in the same java namespace, or a child namespace.

Should each xsd and xjb be in a separate maven module? (I'd rather have them in one maven module, if possible)

When selecting what to compile - how can I specify that `Main.xsd` and `Main.xjb` are one thing and `Common.xsd` and `Common.xjb` another? Do I need to tell the plugin that `Main.xjb/Main.xsd` include `Common.xsd/Common.xjb`?

I guess this has something to do with [Modular Schema Compilation](https://github.com/highsource/jaxb-tools/wiki/Modular-Schema-Compilation) and/or [Using Episodes](https://github.com/highsource/jaxb-tools/wiki/Using-Episodes). But the links are broken.

I marked the broken links with red:
![image](https://github.com/highsource/jaxb-tools/assets/16334222/1060c182-f98f-48c5-8a0c-9754181cba73)

![image](https://github.com/highsource/jaxb-tools/assets/16334222/6cd04429-c6c2-4138-a15d-e953cda0879b)

**Comment by mklemm** [comment]
You don't need modular schema compilation in this case, as long as all of your XSD files are in a common module. Just give the main xsd file to XJC as a parameter, and the binding files as well. You don't specify which xjb belongs to which xsd from the outside, since each binding in the xjb has its own specification as to where it belongs in the schema.
Yours is a very common XJC use case, you should find lots of information about it on the internet.

**Comment by laurentschoelens** [comment]
I've updated some broken links which pointed to old confluence but the pages where still there in this wiki.

I also agree with @mklemm : your example is simple enough not to bring modular schema compilation in action.

**Comment by samodadela** [comment]
With your help I managed to make it work. 

My problem was that I had this in both files. 
```
<jxb:schemaBindings>
            <jxb:package name="org.test.generated.namespace"/>
</jxb:schemaBindings>
```
... this was defaulting to an empty namespace and I was getting:
```
com.sun.istack.SAXParseException2: Multiple <schemaBindings> are defined for the target namespace ""
```
I removed those lines from the child xjb.

I didn't even had to specify `schemaIncludes` and `bindingIncludes`... I just put all the files in the same resources directory and that's it.

Thanks for your help and for fixing the links.
